package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "strip-pragma-loader",
+  "version": "0.0.1",
```

Comment:
We might as well make this `1.0.0`, that way if we ever update it webpack will pick up the changes automatically (`0.x.x` in semver means pre-release and won't auto-update.